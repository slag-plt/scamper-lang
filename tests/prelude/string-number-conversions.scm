(number->string 9)
(number->string 0.4472135954999579)
(number->string 100)
(number->string 4)
(number->string 30.135325698915423)
(string->number "9")
(string->number "0.4472135954999579")
(string->number "100")
(string->number "4")
(string->number "30.135325698915423")