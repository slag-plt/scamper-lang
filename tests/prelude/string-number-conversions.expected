"9"
"0.4472135954999579"
"100"
"4"
"30.135325698915423"
9
0.4472135954999579
100
4
30.135325698915423
